sdk/cosmosdb/cosmos/rollup.config.js

```diff
@@ -2,11 +2,41 @@ import resolve from "rollup-plugin-local-resolve";
 export default [
   {
     input: "dist-esm/index.js",
+    external: [
+      "tslib",
+      "@azure/cosmos-sign",
+      "universal-user-agent",
+      "uuid/v4",
+      "debug",
+      "node-abort-controller",
+      "node-fetch",
+      "atob",
+      "binary-search-bounds",
+      "priorityqueuejs",
+      "semaphore",
+      "crypto-hash",
+      "fast-json-stable-stringify"
+    ],
     output: {
       file: "dist/index.js",
       format: "umd",
       name: "Microsoft.Azure.Cosmos",
-      sourcemap: true
+      sourcemap: true,
+      globals: {
+        "universal-user-agent": "universalUserAgent",
+        "@azure/cosmos-sign": "cosmosSign",
+        "binary-search-bounds": "bs",
+        "crypto-hash": "cryptoHash",
+        "fast-json-stable-stringify": "stableStringify",
+        "uuid/v4": "uuid",
+        "node-abort-controller": "AbortController",
+        "node-fetch": "fetch",
+        tslib: "tslib_1",
```

Comment:
it's really called `tslib_1`?

Reply:
I did not verify. That was the `rollup` suggested name.
I believe they won't be used at all since we are not testing browsers at this point of time.